packages/open-next/src/adapters/config/index.ts

```diff
@@ -31,3 +31,4 @@ export const AppPathsManifestKeys =
   /* @__PURE__ */ loadAppPathsManifestKeys(NEXT_DIR);
 export const MiddlewareManifest =
   /* @__PURE__ */ loadMiddlewareManifest(NEXT_DIR);
+export const AppPathsManifest = loadAppPathsManifest(NEXT_DIR);
```

Comment:
Do we need `/* @__PURE__ */` here?

Reply:
Probably not for all of them, but this is necessary for some part. Without it esbuild fail to remove some of the unused one and it can cause error at runtime